next-env.d.ts

```diff
@@ -0,0 +1,5 @@
+/// <reference types="next" />
```

Comment:
This file can be omitted since it was a typescript declaration file for NextJS however we no longer use next :)